deployment/hasura/metadata/databases/AerieScheduler/tables/public_scheduling_specification_goals.yaml

```diff
@@ -1,3 +1,10 @@
 table:
   name: scheduling_specification_goals
   schema: public
+object_relationships:
```

Comment:
I'm curious that the arrow is going from `scheduling_specification_goals` -> `specification` and not the other way around. What query is this intended to make easier?

Reply:
The UI displays the "goal list" as specification goals so we can include the priority. This gives clients a much easier way to get the full specification goal definition that includes the associated goal, or the associated specification, or any combination of both.

Reply:
That said, we could easily make a relationship the other way if the need arises.